src/main/java/net/glowstone/entity/projectile/GlowProjectile.java

```diff
@@ -59,12 +63,14 @@ public GlowProjectile(Location location) {
 
     @Override
     protected void pulsePhysics() {
-        Vector size = boundingBox.getSize();
-        for (Entity entity : world.getNearbyEntities(
+        if (boundingBox == null) {
```

Comment:
`boundingBox != null` ;)

Reply:
Fixed in 9a17f3d91450a8f9d7970a06dc0da22a1b36b96a.